conformance/conformance_test.cc

```diff
@@ -1337,14 +1376,14 @@ bool ConformanceTestSuite::RunSuite(ConformanceTestRunner* runner,
         WireFormatLite::DecodeFloat(0x7FA12345));
     RunValidJsonTestWithProtobufInput(
         "FloatFieldNormalizeQuietNan", REQUIRED, message,
-        "optional_float: nan");
+        "optional_float: nan", true);
```

Comment:
JSON does not exist for proto2, so there is no need to pass a bool indicating that this is for proto3. Anything JSON-related we can assume is for proto3.